src/main/java/org/reaktivity/nukleus/tcp/internal/reader/stream/StreamFactory.java

```diff
@@ -94,18 +94,10 @@ private Stream(
 
         private int handleStream()
         {
-            try
-            {
-                return handleRead();
-            }
-            catch (IOException ex)
-            {
-                LangUtil.rethrowUnchecked(ex);
-                return 0;
-            }
+            return handleRead();
         }
 
-        private int handleRead() throws IOException
+        private int handleRead()
         {
             assert readableBytes > 0;
```

Comment:
Please remove lines `97-101`, leaving `handleStream` method in place, now that `IOException` is no longer thrown.